{"name": "test_login_page_title[RemoteIE]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <Tests.test_01-LoginLogout.Test_Login object at 0x7fb3af8eea30>\n\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_login_page_title(self):\n        self.loginPage = LoginPage(self.driver)\n>       title = self.loginPage.get_title(data.LOGIN_PAGE_TITLE)\n\nTests/test_01-LoginLogout.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nPages/BasePage.py:31: in get_title\n    WebDriverWait(self.driver, 20).until(EC.title_is(title))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"5d7ce9f7bc399f4a75dbd2090421228edc4151c2\")>\nmethod = <selenium.webdriver.support.expected_conditions.title_is object at 0x7fb3af92a1f0>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "parameters": [{"name": "init_driver", "value": "'RemoteIE'"}], "start": 1613718765314, "stop": 1613718794182, "uuid": "350d644d-4e93-40b8-be16-33972da1fdfd", "historyId": "b18010e547770cffe567adbf39d90af1", "testCaseId": "b98cf488a686d6a295326467119f2fe4", "fullName": "Tests.test_01-LoginLogout.Test_Login#test_login_page_title", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_01-LoginLogout"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "Ishrats-MacBook-Air.local"}, {"name": "thread", "value": "2220-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_01-LoginLogout"}]}